corehq/apps/cloudcare/templates/cloudcare/partials/form_entry/bootstrap5/entry_select.html

```diff
@@ -5,21 +5,20 @@
   <legend class="sr-only" data-bind="text: question.caption()"></legend>
     <div class="sel clear">
       <div data-bind="foreach: choices, as: 'choice'">
-        <div data-bind="css: { checkbox: $parent.isMulti, radio: !$parent.isMulti }, class: $parent.colStyleIfHideLabel">
-          <label>
-              <input data-bind="
-                          checked: $parent.rawAnswer,
-                          checkedValue: $data,
-                          attr: {
-                              id: 'group-' + $parent.entryId + '-choice-' + $index(),
-                              type: $parent.isMulti ? 'checkbox' : 'radio',  {# todo B5: css:checkbox #}
-                              name: $parent.entryId,
-                              class: 'group-' + $parent.entryId,
-                          },
-                      "/>
-              <span data-bind="renderMarkdown: $data, class: $parent.hideLabel ? 'sr-only' : ''"></span>
-          </label>
-          </div>
+        <div class="form-check" data-bind="css: { checkbox: $parent.isMulti, radio: !$parent.isMulti }, class: $parent.colStyleIfHideLabel">
+          <input data-bind="
+                      checked: $parent.rawAnswer,
```

Comment:
do we have a formatter for HTML? This does not look bad. But I cannot figure out what the indentation is based on.

Reply:
I'm not aware of one. it'd be a nice addition.

Reply:
agreed, an html formatter would be great